public/locales/en/status.json

```diff
@@ -14,7 +14,7 @@
   },
   "customApiConfig": "Custom JSON configuration",
   "AskToEnable": {
-    "label": "Help improve this app by sending anonymous usage data."
+    "label": "We have recently updated our telemetry policy to switch to opt-out metrics. You previously had telemetry collection disabled. You can disable telemetry collection (again) on the settings page.",
```

Comment:
@juliaxbow for visibility
cc @tinytb @lidel

Reply:
as another suggestion here we could follow the same verbiage being used on public gateway checker:
"This site collects metrics and other telemetry data in accordance with our metrics collection policy. You can learn more about how we use this information and customize your preferences by clicking “manage settings” (link to settings page).